MobileBuy/buy/src/main/java/com/shopify/buy3/RealGraphCall.java

```diff
@@ -134,11 +137,35 @@ private RealGraphCall(final Query query, final HttpUrl serverUrl, final Call.Fac
 
     checkNotNull(retryHandler, "retryHandler == null");
 
+    if (canceled) {
+      callback.onFailure(new GraphCallCanceledError());
+      return this;
+    }
+
+    final Callback<T> proxyCallBack = new Callback<T>() {
+      @Override public void onResponse(@NonNull final GraphResponse<T> response) {
+        if (canceled) {
+          callback.onFailure(new GraphCallCanceledError());
```

Comment:
To be consistent with sync version of call `com.shopify.buy3.RealGraphCall#execute` that throws `GraphCallCanceledError` in case if call has been canceled, user will be notified about this error in callback.